src/scores/probability/brier_impl.py

```diff
@@ -21,47 +25,205 @@ def brier_score(
+
+    If ``fair_correction`` is set to ``True`` and the number of ensemble members is 1, the
+    fair correction will not be applied to avoid division by zero. If you want to
+    set the minimum number ensemble members required to calculate the Brier score,
+    we recommend preprocessing your data to remove data points with fewer than the
+    minimum number of ensemble members that you want.
 
     Args:
-        fcst: Forecast or predicted variables in xarray.
-        obs: Observed variables in xarray.
+        fcst: Real valued ensemble forecasts in xarray. These values are converted to the
+            probability of the event occurring in this function.
+        obs: Real valued observations variables in xarray. These values are converted to binary
+            values in this function.
+        ensemble_member_dim: The dimension name of the ensemble member.
+        event_thresholds: The threshold(s) that define what is considered an event. If
+            multiple thresholds are provided, they should be monotonically increasing with no NaNs
         reduce_dims: Optionally specify which dimensions to reduce when
             calculating the Brier score. All other dimensions will be preserved.
         preserve_dims: Optionally specify which dimensions to preserve when
-            calculating the Brier score. All other dimensions will be reduced. As a
+            calculating the score. All other dimensions will be reduced. As a
             special case, 'all' will allow all dimensions to be preserved. In
             this case, the result will be in the same shape/dimensionality
             as the forecast, and the errors will be the absolute error at each
             point (i.e. single-value comparison against observed), and the
             forecast and observed dimensions must match precisely.
-        weights: Optionally provide an array for weighted averaging (e.g. by area, by latitude,
-            by population, custom)
-        check_args: will perform some tests on the data if set to True
+        weights: Optionally provide an array for weighted averaging (e.g. by area, by latitude)
+            when calculating the mean score across specified dimensions via ``reduce_dims`` or ``preserve_dims``.
+        fair_correction: Whether or not to apply the fair Brier score correction. Default is True.
+        event_threshold_mode: "The mode to use to define an event relative to a supplied threshold.
+            Default is ``operator.ge``, which means that an event occurs (is forecast)
+            if the observation (forecast) is greater than or equal to the threshold." Passing
+            in ``operator.lt`` will give the same results. Alternatively, you can provide
+            ``operator.gt`` which means that an event occurs (is forecast) if the observation (forecast)
+            is strictly greater than the threshold. Using``operator.le`` will give the same results as ``operator.gt``.
```

Comment:
This could well be my lack of domain expertise here - so if what is written is accurate and clear, then no worries. 

But, as a non-domain expert, here are my two points of confusion:

(1) I am confused how `operator.ge` and `operator.lt` will give the same results. Is what is being looked at symmetrical (I don't what the correct term would be), and is that how they can give the same results? If it is correct as is, then no worries. But I just wanted to check.

(2) I am also confused by the phrase (used twice): "an event occurs (is forecast) if the observation is (forecast)".

Reply:
Thanks very much for those explanations - now I understand :-).